src/appleseed/renderer/modeling/project/projectfileupdater.cpp

```diff
@@ -1834,6 +1835,29 @@ namespace
             ParamArray& frame_params = frame->get_parameters();
             frame_params.strings().remove("save_extra_aovs");
         }
+
+        // Move generic_frame_renderer::passes to the root configuration.
+        void update_passes_path()
+        {
+            for (each<ConfigurationContainer> i = m_project.configurations(); i; ++i)
```

Comment:
This is perfectly fine and you can leave it like that. Note however that in new code we preer to use C++11's range-based `for` loops:

    for (Configuration& config :  m_project.configurations())

Reply:
I prefer too.